File: javascriptAssignments/fizzbuzz.js

function fizzBuzz(num) {
    if (num % 3 == 0 && num % 5 == 0)
        return 'Fizz Buzz';
    else if (num % 3 == 0)
        return 'Fizz';
    else if (num % 5 == 0)
        return 'Buzz';
    else
        return 'Invalid input'
}

console.log(fizzBuzz(15))

File: Fun/python/fizzbuzz.py
def fizzBuzz():
    num1 = int(input("Please enter a whole number: "))
    if num1 % 3 == 0:
        print ("Fizz")
    elif num1 % 5 == 0:
        print ("Buzz")
    elif num1 % 3 == 0 and num1 % 5 == 0: 
        print ("Fizz Buzz")
    else:
    print ("No Fizz Buzz")

fizzBuzz()
